projectroles/migrations/0026_delete_category_settings.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.18 on 2023-03-14 10:21
+
+from django.db import migrations
+
+def clean_up_app_settings(apps, schema_editor):
+    Project = apps.get_model('projectroles', 'Project')
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+
+    # Get all projects that are of type PROJECT_TYPE_CATEGORY
+    category_projects = Project.objects.filter(type='CATEGORY')
+
+    # Loop through all category projects and delete their app settings
+    for project in category_projects:
+        app_settings = AppSetting.objects.filter(project=project)
```

Comment:
We can simply do `filter(project__type=PROJECT_TYPE_CATEGORY)` here, so we can avoid having to query for Project objects separately.